lib/cdo/optimizer.rb

```diff
@@ -78,6 +78,8 @@ def self.cache_key(data)
 
   # ActiveJob that optimizes an image using ImageOptim, writing the result to cache.
   class OptimizeJob < ActiveJob::Base
+    self.queue_adapter = :async
```

Comment:
I'm not as familiar with this and was surprised to see an existing `ActiveJob` instantiation. This makes sense to force this back to the prior default adapter. I'm interested if this is worth revisiting since they are using `perform_later` to create these jobs and assuming the original intention was to potentially batch them... why not do that now that it is possible?

Outside of the scope of this PR.

Reply:
this may be possible, but it does not look like a freebie. things broke spectacularly when we unintentionally changed this to use delayed job: https://codedotorg.slack.com/archives/C03DBDN67B7/p1696946868857699